interactions/console.py

```diff
@@ -1,52 +1,75 @@
 import os
+
 class Console:
 
-    def clear_view(): # czyszczenie ekranu TYLKO NA LINUXS!!!
+    @staticmethod
+    def is_int(stri):  # czy można zmienić w int
+        try:
+            a = int(stri)
+            return True
+        except ValueError:
+            return False
+
+    @staticmethod
+    def clear_view():  # czyszczenie ekranu TYLKO NA LINUXS!!!
         os.system('clear')
 
-    def output (text): # Print w funkcji
+    @staticmethod
+    def output(text):  # Print w funkcji
         print(text)
 
-    def input_text (entry): # Zamiana na string i return
-        text = str(entry)
+    @staticmethod
+    def input_text(entry):  # Wzięcie string i return
+        text = input(str(entry))
         return text
 
-    def input_number (stri,rang): # czy jest to liczba w  liście
-        try:
-            numer = int(stri)
-            is_in_range = False
-            for x in rang:
-              if x == numer:
-                    is_in_range = True
-            if is_in_range == True:
-                return numer
+    @staticmethod
+    def input_number(stri, rang):  # Czy jest to liczba w  liście
+        entry = input(stri)
+        while True:
+            if Console.is_int(entry):
+                numer = int(entry)
+                is_in_range = False
+                for x in rang:
+                    if x == numer:
+                        is_in_range = True
+                if is_in_range:

```

Comment:
tę pętle powyżej można usunąć. wystarczy wykorzystać operator in, 

+ np:  if numer in rang